project/Http4sPlugin.scala

```diff
@@ -297,5 +297,5 @@ object Http4sPlugin extends AutoPlugin {
   lazy val specs2Scalacheck                 = "org.specs2"             %% "specs2-scalacheck"         % specs2Core.revision
   lazy val tomcatCatalina                   = "org.apache.tomcat"      %  "tomcat-catalina"           % "9.0.6"
   lazy val tomcatCoyote                     = "org.apache.tomcat"      %  "tomcat-coyote"             % tomcatCatalina.revision
-  lazy val twirlApi                         = "com.typesafe.play"      %% "twirl-api"                 % "1.3.14"
+  lazy val twirlApi                         = "com.typesafe.play"      %% "twirl-api"                 % "1.3.15"
```

Comment:
Is there a build info published by this project so we could DRY this up with plugins.sbt?  I can't find one.